tests/shellcheck/shellcheck.bzl

```diff
@@ -0,0 +1,14 @@
+load("//tests:inline_tests.bzl", "sh_inline_test")
+
+def shellcheck(name, args, data, sh_flavor="bash", visibility=None):
+  sh_inline_test(
+    name = "shellcheck-%s" % name,
```

Comment:
Similar all the other uses of `%` formatters. Those are not valid in python3, so we shouldn’t use them in bazel/starlark as well, especially since `.format` exists.

Reply:
I changed this to `name = name` in a later commit, as shellcheck was already a prefix of the test's name.